backend/catcall/views.py

```diff
@@ -2,7 +2,7 @@
 from proxy.views import proxy_view
 
 def petfinder_view(request, path):
-    requests_args = {'params': {'key': PETFINDER_API_KEY, 'format':'json', 'animal': 'cat'}}
+    requests_args = {'params': {'key': PETFINDER_API_KEY, 'format':'json', 'animal': 'cat', 'count': 12 }}
```

Comment:
You should paginate instead of just limiting to 12 cats. It's fine for now but write a ticket for pagination if you don't already have one.

Reply:
Lost in my multiple PR whirlwind: I've temporarily just defaulted the response count to 12. The next thing i'm looking into now is adding lazy loading/an infinite scroll for the cat list component so it doesn't have to wait for everything to load before it renders. If infinite scroll ends up being too much of a pain in the ass with available React libraries for it, then i'll just paginate regularly.